pkgs/applications/misc/emoji-picker/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+buildGoModule rec {
+  pname = "emoji-picker";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "bcongdon";
+    repo = "ep";
+    rev = version;
+    hash = "sha256-ElUsmuJ43kOsu4cGvNytM+xHTfuzMo0jcG8Z1cIeHJs=";
+  };
+
+  vendorHash = "sha256-NT7dPMalXpTDaC3R5zWoA7xj6TvTahfo+hIZ2hRPjJg=";
+
+  meta = with lib; {
+    description = "A CLI Emoji Picker";
+    homepage = "https://github.com/bcongdon/ep";
+    license = licenses.mit;
+    maintainers = with maintainers; [ urandom ];
+    mainProgram = "ep";
```

Comment:
```suggestion
    mainProgram = "ep";
    # build just fails, not sure why
    broken = stdenv.isDarwin;
```

Reply:
went ahead and found bug, added a patch, and made an upstream change request; please advise